packages/gatsby-plugin-treat/package.json

```diff
@@ -14,8 +14,7 @@
     "treat": "^1.1.7"
   },
   "dependencies": {
-    "babel-plugin-treat": "^1.1.0",
-    "mini-css-extract-plugin": "^0.8.0"
```

Comment:
If we're importing this package directly, it needs to be defined as a dependency, otherwise it's not guaranteed to be available to us.

Reply:
I’ve reverted the change to resolve that concern. The Next.js plugin doesn’t directly depend on `mini-css-extract-plugin` after all.